Gemfile

```diff
@@ -26,7 +26,7 @@ gem 'geosparql_to_geojson', '~> 0.2'
 gem 'parliament-routes', '~> 0.6'
 
 # Parliament-Utils gem for generic set up and configuration
-gem 'parliament-utils', '~> 0.6', require: false
```

Comment:
Just an FYI, there's no need to update this version number as 0.8 is greater than 0.6 but less than 1.0